src/mono/mono/profiler/browser.c

```diff
@@ -258,6 +271,9 @@ parse_arg (const char *arg)
 		char *end;
 		desired_sample_interval_ms = strtod (val, &end);
 	}
+	else if (match_option (arg, "stacks", NULL)) {
```

Comment:
Maybe provider=sample and provider=devtools

like:

provider=sample,callspec=all,interval=5

provider=devtools will be default if not specified for backward compability.

Just an idea.

Reply:
or if sample is redundant then "providers" could be eventpipe and devtools.